firebaseai/FirebaseAIExample/ImagenExample/ImagenViewModel.swift

```diff
@@ -42,13 +42,13 @@ class ImagenViewModel: ObservableObject {
   private var generateImagesTask: Task<Void, Never>?
 
   init(firebaseService: FirebaseAI) {
-    let modelName = "imagen-3.0-generate-002"
+    let modelName = "imagen-4.0-fast-generate-001"
     let safetySettings = ImagenSafetySettings(
       safetyFilterLevel: .blockLowAndAbove
     )
     var generationConfig = ImagenGenerationConfig()
     generationConfig.numberOfImages = 4
-    generationConfig.aspectRatio = .landscape4x3
+    generationConfig.aspectRatio = .square1x1
```

Comment:
I switched this to 1:1 since the images are square in the UI and looked stretched:
https://github.com/firebase/quickstart-ios/blob/c39fe8da816c408687a6425c1ed9da8deabd0afe/firebaseai/FirebaseAIExample/ImagenExample/ImagenScreen.swift#L60